pvr.iptvsimple/changelog.txt

```diff
@@ -1,3 +1,6 @@
+v22.1.2
+- Fix #KODIPROP parsing from playlists
+
```

Comment:
Is `Fix wrong fix to manifest user-agent header for inputstream.adaptive` missing from changelog / commits? Or is it not required for Piers branch?

Reply:
piers its ok, it is only on omega that i accidentally cherry-picked the old commit without realizing it

Reply:
Ok, so any change needed for Omega?

Reply:
you have already merged the omega backport that contains reverted "bad" commit and applied the correct one